File: template-php-lumen/tests/HelloTest.php
<?php

namespace Tests;

use PHPUnit\Framework\TestCase;
use Rogue\Hello;

class HelloTest extends TestCase {

    /**
     * @test
     */
    public function shouldGreet() {
        $hello = new Hello();

        $name = $hello->getName();

        $this->assertEquals("World!", $name);
    }

}

File: ohgod/src/test/java/org/codecop/rogue/template/HelloTest.java
package org.codecop.rogue.template;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.codecop.rogue.template.Hello;
import org.junit.jupiter.api.Test;

class HelloTest {

    Hello hello = new Hello();

    @Test
    void get() {
        assertEquals("World!", hello.getName());
    }

}
